packages/clarity-js/types/data.d.ts

```diff
@@ -404,7 +406,7 @@ export interface UpgradeData {
 }
 
 export interface ExtractData {
-    [key: string]: string | number;
+    [key: number]: [string, string][];
```

Comment:
What do the two element in the tuple (array) represent? Suggest you declare a type for the pair separately to make the intent clearer.